.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-sudo: false
+os: linux
```

Comment:
I checked this is only appearing as `info` not as `warning` so I think we can reverse it for now. 
(The thing is that these things keep changing, so it's not worth our time to keep the `info` part changing, only when it appears as a `warning` that we need to do something about it! :smile: )